tgui/packages/tgui/interfaces/Instrument.js

```diff
@@ -0,0 +1,377 @@
+/* eslint-disable max-len */
```

Comment:
hmmm

Reply:
I didn't want to put that in but had to because of the help menu, otherwise the 80 character per line limit should be respected everywhere else.

Reply:
TG has this
```jsx
    <Section title="Business Exploration Protocol Incubation Sink">
      <Section 
        title="Information"
        backgroundColor="#450F44"
        buttons={(
          <Button
            icon="power-off"
            content={data.manual_power ? 'Off' : 'On'}
            selected={!data.manual_power}
            onClick={() => act(ref, 'toggle_power')} />
        )}>
        All you need to know about the B.E.P.I.S. and you!
        The B.E.P.I.S. performs hundreds of tests a second
        using electrical and financial resources to invent
        new products, or discover new technologies otherwise
        overlooked for being too risky or too niche to produce!
      </Section>
```
You can newline text without needing to go above the 80 character limit

Reply:
I'm aware. Adding newlines results in this (note the missing spaces in some places):

![image](https://user-images.githubusercontent.com/1496804/87798220-037fd380-c84c-11ea-9d48-6cded3245210.png)

I guess it *can* be fixed by adding `&nbsp;`:

![image](https://user-images.githubusercontent.com/1496804/87798757-b05a5080-c84c-11ea-8a86-5247b9b40a94.png)

![image](https://user-images.githubusercontent.com/1496804/87798734-aafd0600-c84c-11ea-9025-1eda0d479e9e.png)

Reply:
I would much rather this than having to break the linter's line limiting